llvm/include/llvm/CodeGen/RegisterUsageInfo.h

```diff
@@ -66,6 +63,52 @@ class PhysicalRegisterUsageInfo : public ImmutablePass {
   const LLVMTargetMachine *TM = nullptr;
 };
 
+class PhysicalRegisterUsageInfoWrapperLegacy : public ImmutablePass {
```

Comment:
Certain passes are ported with `WrapperPass` and others with `WrapperLegacy`. We should be consistent with it.
I think the initialize function of the former will have **initialize****WrapperPassPass** which looks odd with Pass doubled at the end. The latter seems better in that way.

Reply:
Yes, should be Wrapper

Reply:
Yes, I am naming analysis passes I am porting with Legacy.